tests/units/aps/test_aps_parser.py

```diff
@@ -165,59 +158,60 @@ def parsed_articles(parser, articles):
         ),
         pytest.param(
             [
-                "http://link.aps.org/licenses/aps-default-license",
-                "http://link.aps.org/licenses/aps-default-license",
+                [{"url": "http://link.aps.org/licenses/aps-default-license"}],
+                [{"url": "http://link.aps.org/licenses/aps-default-license"}],
             ],
-            "licenses",
-            id="test_licenses",
+            "license",
+            id="test_license",
         ),
         pytest.param([[], []], "field_categories", id="test_field_categories"),
-        pytest.param(
-            [
-                {
-                    "files": [
-                        {
-                            "url": "http://harvest.aps.org/v2/journals/articles/{0}".format(
-                                "10.1103/PhysRevA.103.042607"
-                            ),
-                            "headers": {"Accept": "application/pdf"},
-                            "name": "{0}.pdf".format("10.1103/PhysRevA.103.042607"),
-                            "filetype": "pdf",
-                        },
-                        {
-                            "url": "http://harvest.aps.org/v2/journals/articles/{0}".format(
-                                "10.1103/PhysRevA.103.042607"
-                            ),
-                            "headers": {"Accept": "text/xml"},
-                            "name": "{0}.xml".format("10.1103/PhysRevA.103.042607"),
-                            "filetype": "xml",
-                        },
-                    ]
-                },
-                {
-                    "files": [
-                        {
-                            "url": "http://harvest.aps.org/v2/journals/articles/{0}".format(
-                                "10.1103/PhysRevB.103.165408"
-                            ),
-                            "headers": {"Accept": "application/pdf"},
-                            "name": "{0}.pdf".format("10.1103/PhysRevB.103.165408"),
-                            "filetype": "pdf",
-                        },
-                        {
-                            "url": "http://harvest.aps.org/v2/journals/articles/{0}".format(
-                                "10.1103/PhysRevB.103.165408"
-                            ),
-                            "headers": {"Accept": "text/xml"},
-                            "name": "{0}.xml".format("10.1103/PhysRevB.103.165408"),
-                            "filetype": "xml",
-                        },
-                    ]
-                },
-            ],
-            "extra_data",
-            id="test_extra_data",
-        ),
+        # THIS IS A TEST FOR EXTRA DATA
```

Comment:
we shouldn't leave comments :)

Reply:
There is one more comment. It was just a question and reminder about extra data 😅:
https://github.com/cern-sis/workflows/blob/full_aps_proccessing/dags/aps/parser.py#L73-L74

Reply:
Deleted comments